webapp/store/auth.js

```diff
@@ -18,6 +19,9 @@ export const mutations = {
   SET_PENDING(state, pending) {
     state.pending = pending
   },
+  SET_HAS_AGREED_TO_LATEST_TERMS_AND_CONDITIONS(state, boolean) {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-16T08:55:54Z" title="Friday, August 16th 2019, 10:55:54 am +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -18,6 +19,9 @@ export const mutations = {
   SET_PENDING(state, pending) {
     state.pending = pending
   },
+  SET_HAS_AGREED_TO_LATEST_TERMS_AND_CONDITIONS(state, boolean) {
```


Remove, this is not necessary.